doc/release/1.13.0-notes.rst

```diff
@@ -29,6 +29,12 @@ Future Changes
 Compatibility notes
 ===================
 
+
+Error type changes
```

Comment:
You've reminded me that I should add another entry under this heading regarding #8584, so thanks for that